packs/nagios/etc/st2service_handler.py

```diff
@@ -5,16 +5,24 @@
     import simplejson as json
 except ImportError:
     import json
-import requests
+import os
 import sys
-import urlparse
+
+import requests
```

Comment:
I was wondering about that as well - it looks like there isn't, but requests was already used before that change.

If nothing else, we need to mention it in the docs (if we don't do that already).